generator.go

```diff
@@ -258,7 +262,12 @@ func (g *Gen) getClockSequence() (uint64, uint16, error) {
 	g.storageMutex.Lock()
 	defer g.storageMutex.Unlock()
 
-	timeNow := g.getEpoch()
+	var timeNow uint64
+	if useUnixTSMs {
+		timeNow = uint64(g.epochFunc().UnixMilli())
```

Comment:
https://github.com/gofrs/uuid/pull/104 glad we're going back to this newer, easier to understand verbiage.